win32ss/drivers/videoprt/dispatch.c

```diff
@@ -805,7 +805,9 @@ IntVideoPortDispatchDeviceControl(
 
         case IOCTL_VIDEO_IS_VGA_DEVICE:
             WARN_(VIDEOPRT, "- IOCTL_VIDEO_IS_VGA_DEVICE is UNIMPLEMENTED!\n");
-            Status = STATUS_NOT_IMPLEMENTED;
+            *((PBOOLEAN)Irp->AssociatedIrp.SystemBuffer) = IntIsVgaSaveDriver(DeviceObject);
```

Comment:
Is the size of the SystemBuffer already checked, to be sure it indeed can store one byte (a BOOLEAN) ?